### DISCUSSION TITLE
2007/10/17/%E4%B9%B0%E6%88%BF%E7%9A%84%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#182] [Announcements]
# 2007/10/17/%E4%B9%B0%E6%88%BF%E7%9A%84%E7%BB%8F%E5%8E%86

也许最近房价会跌，那是我造成的:)， 我最近买了套房子。我一项没什么财气，买什么什么就跌价，不买的就一个劲猛涨。

https://blog.qizhen.xyz/2007/10/17/%E4%B9%B0%E6%88%BF%E7%9A%84%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Apple
2007年10月29日 19:54 编辑
楼主的建议很不错哦,加油~~~~ :)
 

回复
奕瑋
2007年10月17日 21:38 编辑
又是沙发吗？
恭喜阮主成为业主。今年早些时候，您是又娶媳妇又过年，年底又成为有房有车，真是喜事连连啊！
明年争取生个奥运宝宝吧，到时候您享受一下初为人父的滋味，再写一篇日志“当爹的经历”。

回复
